src/components/dashboard/FeaturedApps/FeaturedApps.tsx

```diff
@@ -23,14 +23,14 @@ const StyledGridItem = styled(Grid)`
 `
 
 export const FeaturedApps = (): ReactElement | null => {
-  const [allApps = [], , isLoading] = useRemoteSafeApps()
```

Comment:
Why replace this with `useSafeApps`? It doesn't look like any non-remote apps are used

Reply:
I grepped, and it was the only place where this hook was used apart from `useSafeApps`.
I think it's better if `useSafeApps` is the only entry to safe apps, and the remote apps hook should remain as a raw fetcher.

Reply:
> I grepped, and it was the only place where this hook was used apart from `useSafeApps`. I think it's better if `useSafeApps` is the only entry to safe apps, and the remote apps hook should remain as a raw fetcher.

`useSafeApps` wasn't meant to be the only entry point, it just happens that most of the places need apps from multiple sources 🤷 If you know that you only need the remote apps, in my opinion, it's better to use the raw hook because `useSafeApps` also requests other resources which are unnecessary in this case